### PR TITLE
Add slackin to the index page and also to the people section

### PR DESCRIPTION
**Pull request by thomet**
We have now one slack group for all onruby members/groups. I added a
invite box/badge, provided by slackin, to the application because it's only
possible to join the slack group by an invite.

screen shot 2015-10-22 at 12 33 21

### CODE REVIEW

**Comment by phoet** [comment]
i'd like to discuss this as it will affect all usergroups.

PTAL @kikito @titanoboa @PragTob @jhilden 

my preference would be to have this as a configurable on the usergroups

**Comment by thomet** [comment]
My intention is that we have one slack for all youser groups. That's why I don't make this configurable. Each usergroup can have it's own channel. But then we have also all members in a general channel which brings us more together. It's good to share knowhow, ask question and to organise events together.

**Comment by bitboxer** [comment]
The thing is that Berlin already has his own slack thing going on. And I don't think we will move that to somewhere else.

**Comment by kikito** [comment]
I don't think we need this for Madrid; Our community just doesn't seem to generate enough web traffic for a slack channel - the mailing list, for example is relatively quiet: 10 mails per week in a specially good week.

If you want to add this for everyone, it's ok. I am just not sure we'll use it.

**Comment by thomet** [comment]
@kikito I'm sure all ruby devs (independently from his user group) will have an effort from this channel when there are other ruby devs

@phoet I suggest, that each user group can configure his own slackin. And user group there like to have on slack for all, can configure the onruby slackin. And the other ones how will like to be private or sperated, can configure is own. Should I configure a fallback to onruby? And should I remove the slack invite from the global index page?

**Comment by PragTob** [comment]
As @bitboxer mentioned berlin already has its own slack (well with more friends from all over the world). Having it configurable would be nice, not sure if we're using slackin. But yeah would definitely want to change/deactivate it.

Also personally for the index page I'd move it further down - onruby is primarily about the user groups and then maybe secondarily about the slack group :)

Anyhow, nice idea - thank you. We want to have a more prominent place to put our berlin slack (right now it's burried in emails and occasionally mentioned on twitter).

File: app/assets/stylesheets/application.sass [hunk 1]
```diff
@@ -175,6 +175,19 @@ header
...
       +border-radius(0)
+    .slackin
+      height: 250px
+      width: 320px
+      overflow: hidden
+      position: relative
+      margin-left: auto
+      margin-right: auto
+      iframe
+        position : absolute
```
Comment (wteuber) [inline app/assets/stylesheets/application.sass]: useless whitespace "position` `:"

**Comment by sangyye** [comment]
Hey there, we from colognerb have also our own slack and slackin running and we would like to have this feature to make our slack visible. Thanks, for the idea.

**Comment by phoet** [comment]
@thomet would you like to change your proposal to a configurable solution?

* [ ] remove global slack link on www.onruby.de
* [ ] add an option to the [Usergroup](https://github.com/phoet/on_ruby/blob/master/app/models/usergroup.rb) class and set [configurations to whitelabel.yml](https://github.com/phoet/on_ruby/blob/master/config/whitelabel.yml)
* [ ] hide the slack icon when nothing configured per usergroup

**Comment by thomet** [comment]
Currently I haven't time for that, but when I have time, I will do that.

**Comment by thomet** [comment]
All task done :D

**Comment by wteuber** [comment]
:+1:

**Comment by phoet** [comment]
the travis docker build is currently broken and i think it's actually not doing what we want, so i'm going to fix that first (or remove the docker setup from travis for now)

**Comment by phoet** [comment]
and its live :dancers: thx again

**Comment by bitboxer** [comment]
Oh, sorry @phoet . I just accidently pushed to the wrong repo. Wanted to do a PR with the slack for rug:b :sob: . Hope it is okay.

**Comment by phoet** [comment]
No worries, just have a look at the small change i made, you will probably
have to provide the full URL to the script.

Bodo Tasche <notifications@github.com> schrieb am Fr., 15. Jan. 2016, 08:30:

**Pull request merged**